settings.gradle

```diff
@@ -19,6 +19,7 @@ include 'fabric-blockrenderlayer-v1'
 include 'fabric-commands-v0'
 include 'fabric-containers-v0'
 include 'fabric-content-registries-v0'
+include 'fabric-content-registries-v1'
```

Comment:
Needs to be renamed to `fabric-content-registry-api-v1` to be in line with new naming standards for modules.

Reply:
Actually, per the RTC issue, this would be -registries. Registry-api sounds like an API for adding registries

Reply:
Hmm I see that entry after I looked again at the RFC